contracts/SSVNetwork.sol

```diff
@@ -286,11 +288,7 @@ contract SSVNetwork is OwnableUpgradeable, ISSVNetwork {
             }
         }
 
-        if (amount > 0) {
-            _deposit(msg.sender, operatorIds, amount.shrink());
-            pod.balance += amount.shrink();
-        }
-
+        pod.balance += amount.shrink();
```

Comment:
Im pretty sure that if you send an amount of 0 this will cause the tx to fail right? shrink() calls shrinkable()

  function shrinkable(uint256 value) internal pure returns (uint256) {
    require(value % DEDUCTED_DIGITS == 0, "Precision is over the maximum defined");
    return value;
  }